Formula/docker-ls.rb

```diff
@@ -0,0 +1,31 @@
+class DockerLs < Formula
+  desc "Tools for browsing and manipulating docker registries."
+  homepage "https://github.com/mayflower/docker-ls"
+  url "https://github.com/mayflower/docker-ls.git",
+    :tag => "v0.3.1",
+    :revision => "d80310976c9707e261e57ebfa9acf4e0b1781460"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+
+    (buildpath/"src/github.com/mayflower/docker-ls").install buildpath.children
+
+    system "go", "generate", "github.com/mayflower/docker-ls/lib"
+    system "go", "install", "github.com/mayflower/docker-ls/cli/..."
+
+    bin.install buildpath/"bin/docker-ls"
+    bin.install buildpath/"bin/docker-rm"
+  end
+
+  test do
+    assert_match /\Wlatest\W/m, pipe_output("#{bin}/docker-ls tags \
```

Comment:
My preferred style is [this sort of thing](https://github.com/Homebrew/homebrew-core/blob/30abddeed8f2b8d6e3d7b5ee313ddcd2c8ff1671/Formula/ired.rb#L20-L27) if that works here, and if I'm not being a complete PITA 😉. ILZ can overrule me on this though if desired.

Reply:
I must admit that I honestly don't know how to apply this to my usecase: replacing the argument of `pipe_output` with a multiline herestring breaks.

Reply:
Let's leave it for now & see how much @ilovezfs cares. If he's cool with it then 🤷‍♂️.